holoviews/plotting/widgets/widgets.js

```diff
@@ -139,8 +129,8 @@ SelectionWidget.prototype.get_key = function(current_vals) {
   var key = "(";
   for (var i=0; i<this.slider_ids.length; i++)
   {
-    val = this.current_vals[i];
-    if (!(typeof val === 'string')) {
+    let val = this.current_vals[i];
```

Comment:
Should probably still be ``var``. I am strongly in favor of switching to ``let`` and ES6 but not in this PR!

Reply:
Right, that was accidental.